tests/plugins/chart/basic_chart.test.ts

```diff
@@ -1723,6 +1723,48 @@ describe("Chart evaluation", () => {
     ).toBe("#REF");
   });
 });
+
+describe("Cumulated Data line chart", () => {
+  test.each([
+    {
+      dataSets: ["B2:D7"],
+      InitialData: [11, 12, 13, "P4"],
+      expectedCumulativeData: [11, 23, 36, "P4"],
+    },
+    {
+      dataSets: ["B7:D7"],
```

Comment:
this second dataset is not necessary. My point was that you should at least test the fact that the aggregator is passing non numerical values and doesn't crash when it finds one. which is done in the previous dataset.